src/ReactWrapper.jsx

```diff
@@ -204,10 +208,13 @@ class ReactWrapper {
    * @returns {ReactWrapper}
    */
   unmount() {
+    const { component } = this;
+    const inst = component._reactInternalInstance._renderedComponent.getPublicInstance();
     if (this.root !== this) {
       throw new Error('ReactWrapper::unmount() can only be called on the root');
     }
     this.single('unmount', () => {
+      instanceMap.set(component, inst);
       this.component.setState({ mount: false });
```

Comment:
Yeah it's destructured out at the top of the function. The next line should actually be `component` too.